### DISCUSSION TITLE
2024-02-21 Weekly Meeting 147 Notes/Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#447] [Weekly Meetings]
Weekly Meeting 147 is happening on Wednesday, 21 February, 2024 from 9:00 AM PDT (17:00 UTC) - 10:00 AM PDT (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting in the NEW SERVER, in the omi-weekly-meeting channel of the [OMI Discord](https://discord.gg/rrHfT7H3).
We regularly use the # omi-meeting-chat channel for text chat.

### Agenda
- #446 
- #445 
- #438 
  - @mrmetaverse found the w3c blog and some next steps
- adding tix to the backlog https://github.com/omigroup/media/discussions/70#discussioncomment-8469289
- Getting the blog running
  - blog about the logo and how we interpret it (very rough draft, based on the conversation on Discord https://hackmd.io/@indiebio/omi-logo)
  - blog about the new discord 
  - blog about the new chairs and thanks prev. 

Working group check ins
- glTF
- publicity
-  backlog meeting
Demos and discussion

**Comment by indiebio** [comment]
publicity team - mock up of new version of website
antpob did a great job on the website and the design. Great to continue this work.
aaron - audio clip glTF demo
Thomukas1 demo interoperable avatar WIP

**Comment by indiebio** [comment]
Website feedback here!

**Reply by indiebio** [reply]
more "few of our friends"
 - M3, MSF, W3C, OMA?

**Reply by indiebio** [reply]
new logo is integrated.
Have a "see a few of our members' projects, linking to the awesomeness directory or logos that link to projects directly, e.g. the Virtual Worlds museum etc
Great work! Continue the work before vote / make live

**Reply by indiebio** [reply]
Jimmy: Ref the discussions on website updates please pipe in add your thoughts here to the github we will link it to the website repo as we go .....  💫 
https://github.com/omigroup/media/discussions/75

**Comment by indiebio** [comment]
fun finds etc
new Apple headset VisionPro - nothing wow, but the best currently on the market. Compares to Hololens - SoloSolipsist. Flow between different devices is good.

**Reply by indiebio** [reply]
Sifr - https://www.theregister.com/2024/02/15/apple_vision_pro_returns/

**Reply by indiebio** [reply]
https://x.com/dankvr/status/1760315196076077560?s=20 - Blender VR plug in

**Comment by indiebio** [comment]
Thomukas demo - VRMs change everything! V-Sekai Godot plugin: https://github.com/V-Sekai/godot-vrm

**Reply by mrmetaverse** [reply]
announced on twitter https://x.com/open_metaverse/status/1760357602829971864?s=20

**Reply by mrmetaverse** [reply]
![GG4KvHZWQAAVsgf](https://github.com/omigroup/omigroup/assets/63426722/79e5172f-94e3-430d-b280-3667e073df31)

shoutout to @fire and the entire V-Sekai team.

**Comment by mrmetaverse** [comment]
@aaronfranke shared their usage of various OMI protocols and glTF extensions. Announced here on [twitter](https://twitter.com/open_metaverse/status/1760358884068163935/photo/4).

![GG4MG15XUAA3bH0](https://github.com/omigroup/omigroup/assets/63426722/7ea788a8-4d57-4515-b7c0-4a978bb775b1)
![GG4MzAhWEAMhFps](https://github.com/omigroup/omigroup/assets/63426722/b7699a01-21a4-4519-8506-48a88c277cd5)

**Reply by indiebio** [reply]
aaron glTF audio asset update: not just audio, also an asset integrating the physics. High quality. 

Also, looking at the code, there is now a single-root plugin that makes things easier for integrations across game engines.

KHR_audio_emitter sample boom box: https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/KHR_audio_emitter/examples/boom_box

**Reply by indiebio** [reply]
shared on Twitter: Like and share
https://twitter.com/open_metaverse/status/1760358884068163935
https://twitter.com/open_metaverse/status/1760357602829971864

**Comment by indiebio** [comment]
SoloSolopsist: AVP Avatar demo: https://www.twitch.tv/videos/2069524840/

**Comment by mrmetaverse** [comment]
## Blog 

How should we approach the blogging? 

We have these ideas for topics:

We think in this order: 
- new chairs
- new logo 
- new discord 
- first newsletter update combining those three
- Future
  - new protocols/tests 
  - highlight members and their projects

We can publish through these channels: 
* W3C Mailing List and Page 
* Our Own Website 
* Twitter 
* Is there another?
